std/path/globrex.ts

```diff
@@ -40,11 +45,6 @@ export interface GlobrexResult {
  * Convert any glob pattern to a JavaScript Regexp object
  * @param glob Glob pattern to convert
  * @param opts Configuration object
- * @param [opts.extended=false] Support advanced ext globbing
- * @param [opts.globstar=false] Support globstar
- * @param [opts.strict=true] be laissez faire about mutiple slashes
- * @param [opts.filepath=""] Parse as filepath for extra path related features
```

Comment:
The default is actually `false` and not `""`. This is a confusing property name.